app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1945,10 +1945,6 @@ class BrowserTabViewModel(
         }
     }
 
-    fun onQuickAccesItemClicked(it: SavedSite) {
-        command.value = SubmitQuery(it.url)
```

Comment:
I think we can remove `SubmitQuery` from the commands, doesn't look like it's been used.